src/ReleaseHistory.md

```diff
@@ -2,6 +2,7 @@
 
 ## Unreleased
 
+* BUGFIX: Update command line argument name from `postUri` to `post-uri`. [#2427](https://github.com/microsoft/sarif-sdk/pull/2427)
```

Comment:
>BUGFIX [](http://example.com/codeflow?start=2&length=6)

Why is this categorized as a bugfix? Did the naming cause a problem for customers? Was this to address a particular issue?

Reply:
this is created from the issue
[https://github.com/microsoft/sarif-sdk/issues/2428](https://github.com/microsoft/sarif-sdk/issues/2428)

Did the naming cause a problem for customers
---- no, because it was not yet released.

Why is this categorized as a bugfix
---- since it was not yet released, so change of the parameter name will not break user. let me know if you think I should use another keyword.

Reply:
If the issue hasn't shipped as a customer issue, you don't have to document it in the release notes. Make sense? In other words, all of these notes should refer to changes strictly from v2.4.12.

Now if it *had* shipped previously, you'd need to clearly mark this as a breaking change.

So the note here is remove the release notes entry. Developers use commit history to track intra-version changes. Release notes, again, only document changes from the last shipped version.

Reply:
use commit history to track intra-version changes ---- got it, yes then I should remove it.